cluster/ci/config/prow/config.yaml

```diff
@@ -3209,6 +3209,33 @@ presubmits:
         - --template=/usr/local/e2e-aws
         - --target=e2e-aws
 
+  - name: shell-check
+    trigger: "(?m)^/shell-check"
+    rerun_command: "/shell-check"
+    context: shell-check
+    always_run: true
+    skip_report: false
+    agent: kubernetes
+    decorate: true
+    spec:
+      containers:
+      - image: quay.io/coreos/shellcheck-alpine:v0.5.0
+        command:
+        - sh
+        args:
+        - -c
+        - 'for file in $(find ./ -type f -name "*.sh");
```

Comment:
This is fine but generally you want to add this as a script in the repository so that developers can run it locally, too

Reply:
@stevekuznetsov @alejovicu we can include shellcheck|terraform-lint|terraform-fmt|go unit tests as targets in a Makefile in the installer repo? then combine them all here with a single make cmd? (eventually, we're replacing the bazel build with Make, too, AFAIK. We can start with a simple Makefile for testing to replace the travis cmds, then add on in near future?

Reply:
> we can include shellcheck|terraform-lint|terraform-fmt|go unit tests as targets in a Makefile in the installer repo? then combine them all here with a single make cmd?

The only downside I see to using a single `make` call here would be granular retesting.  All of these are fast, so I'm in favor of lumping them all into a single Prow test, but I wouldn't want something heavy (like e2e tests) lumped in with them ;).

Reply:
Actually, another issue with lumping them together would be errors in one test masking issues in another.  My preferred approach to that would be to use [`--keep-going`][1], so a failure in one subtest (e.g. Terraform formatting) wouldn't mask errors in another (e.g. Go unit tests).

[1]: https://www.gnu.org/software/make/manual/html_node/Errors.html

Reply:
@wking we had a long discussion on this :) I think @alejovicu @stevekuznetsov @rafael-azevedo and I are all on the same page with this now.. we'll def want to have separate targets.. like 'make tflint, make cli_unit, etc'   and we'll have a hack directory (following convention of other openshift/repos) where we'll keep the simple scripts, like 'tflint.sh', etc